ci/init-plugin-test-service.sh

```diff
@@ -41,3 +41,13 @@ docker exec -i rmqnamesrv /home/rocketmq/rocketmq-4.6.0/bin/mqadmin updateTopic
 
 # prepare vault kv engine
 docker exec -i vault sh -c "VAULT_TOKEN='root' VAULT_ADDR='http://0.0.0.0:8200' vault secrets enable -path=kv -version=1 kv"
+
+# prepare openfunction env
+docker pull apisixtestaccount123/sample-go-func:v1
```

Comment:
I'm wondering if OpenFunction releases some sample functions 🤔.

Reply:
They do， but the functions just print the URI of request, which doesn't meet my test cases(e.g. test body, test headers). Therefore, I just fork and change their functions a little bit.

Reply:
can we expose the Dockerfile for `apisixtestaccount123/sample-go-func:v1`?
 My concern is that when we want to make updates based on this image later, then it will be difficult for us to do so.

Reply:
I ‘ll add local function building process latter，then we just need to update the code of  repositories.